### ISSUE TITLE
ML/network disease gene prediction projects

### ISSUE THREAD

**Issue by damiansm** [whri-phenogenomics/whri-phenogenomics#10]
https://docs.google.com/presentation/d/1fLNvW60h8IAsyZVIWOC6VZhqWtQDWvm_cclnmmdNDP4/edit#slide=id.g22b6b07584c_2_100

**Comment by pilarcacheiro** [comment]
1. Common set of gene annotations and data integration

Cell essentiality data (DepMap)
Mouse viability and phenotypic data (IMPC and MGI) @pilarcacheiro 
Ppi data (STRING)
Pathways (Reactome)
GO annotations
PhenoDigm scores (human, mouse) @damiansm 
Protein complex (CORUM)
Protein families (Panther)
Intolerance to variation metrics (gnomAD)

**Comment by pilarcacheiro** [comment]
- Gene annotations: repository for that (overlap with Morphic annotations, Gabriel also contributing) https://github.com/whri-phenogenomics/gene-annotations/
- Pilar still to add annotations
- NDD: revisiting paper: 75% diagnosis Decipher cohort (difficult cases) were de novo variants - so AD: agreed that we should focus on AD disorders 
- DDG2P & GEL panel NDD & any other gene with HPO annotation of intellectual disability / developmental delay
- Come up with a set of terms for mantis-ml
- String: It might be worth to compare different networks built with different sources of STRING annotations (see Domino paper S table 2) - it is possible to access string files without previous download

**Still on track to complet April objectives 
Objectives for May: 1) Implement the analysis pipeline and 2) Explore preliminary results**

**Comment by pilarcacheiro** [comment]
I have added a README file to help understand the code and output of the new scripts:
https://github.com/whri-phenogenomics/gene-annotations/blob/NDD-genes/scripts/README.md

**Comment by pilarcacheiro** [comment]
Things to explore / keep in mind

ML:
- Missing data and imputation: which % threshold are we going to consider? above which % of missing data should we discard that feature? / Different imputation approaches for RF and XGBoost (pros and cons)
- Cross Validation. Also training / test / validation set
- Imbalance classes - best ways to deal with this with current algorithms (multiple random samples of the negative class of similar size to the positive class seems like a sensible approach)
Network analysis:
- Methods to compare different networks
- Explore different approaches to compute proximity in the network: random walk algorithm (implemented in igraph), distance of each node to an NDD node + statistical framework to compute significance

**Comment by damiansm** [comment]
Phenotype prediction off same data sources?
